app/controllers/listings_controller.rb

```diff
@@ -262,6 +262,17 @@ def create
 
     shape = get_shape(Maybe(params)[:listing][:listing_shape_id].to_i.or_else(nil))
 
+    listing_uuid = UUIDTools::UUID.timestamp_create
+    author_uuid = UUIDTools::UUID.parse_raw(Base64.urlsafe_decode64(@current_user.id))
```

Comment:
I guess we could move the `author_uuid` inside the `if FeatureFlagHelper.feature_enabled?(:availability) ...` branch since it's not used elsewhere?